styles/Checkbox.styled.js

```diff
@@ -0,0 +1,8 @@
+import styled from 'styled-components';
+
+const StyledCheckbox = styled.input`
+	height: 2rem;
+	border-radius: 5px;
+`;
+
+export default StyledCheckbox;
```

Comment:
Why do you name it `StyledCheckbox` when it is just a normal `input` field and not what the name indicates, a `checkbox`

Reply:
This is a styling just for the checkbox. I have other styled inputs as well. I moved this to them.